weac/utils/geldsetzer.py

```
@@ -0,0 +1,165 @@
+"""
+Hand hardness + Grain Type Parameterization to Density
+according to Geldsetzer & Jamieson (2000)
+`https://arc.lib.montana.edu/snow-science/objects/issw-2000-121-127.pdf`
+
+Inputs:
+Hand Hardness + Grain Type
+Output:
+Density [kg/m^3]
+"""
+
+from typing import Tuple
```

Ruff (0.12.2)

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)